### DISCUSSION TITLE
Problem with instalation

### DISCUSSION THREAD

**Discussion by marvays** [elementary/triage#365] [Q&A]
Hi. I am new in linux. I found for first elementary OS. But have problem after instal...

installation appears to be successful. but after restarting and removing the installation USB, it doesn't want to boot.
It says I have nothing installed.
I have legacy turned off because there is no other way to install it.

When I tried reinstalling and everything again, it didn't work anymore. I tried a new install from USB 3 times before it got installed but I always end up here.

**Comment by ealaney** [comment]
I know this question is more than two months old, but any additional detail you can provide on the installation attempts will help determine the issue. Can you describe the computer on which you are installing elementary OS? What kind of drive is the destination drive? Hard drive? SSD? How large is it? When you attempt to boot the system without the USB key, are there any messages displayed before the message about there being nothing installed? What was installed on the computer before you attempted to install elementary OS?
